tests/acceptance/AdminDatasetForm.feature

```diff
@@ -465,14 +472,20 @@ Feature: form to update dataset details
       | "DataAvailableForReview" |
       | "DataPending"            |
 
-  @ok @dataset-status
-  Scenario: Links to create mockup or to open mockup are not present for a published dataset
+  @ok @test
+  Scenario: Links to create mockup is present for a submitted dataset
     Given I am on "/adminDataset/update/id/5"
-    And I select "Published" from the field "Dataset_upload_status"
+    And I select "DataAvailableForReview" from the field "Dataset_upload_status"
     And I press the button "Save"
     When I am on "/adminDataset/update/id/5"
-    Then I should not see "Create/Reset Private URL"
-    And I should not see "Open Private URL"
+    And I select "Submitted" from the field "Dataset_upload_status"
+    And I press the button "Save"
+    When I am on "/adminDataset/update/id/5"
+    Then I should see "Create/Reset Private URL"
+    And I press the button "Create/Reset Private URL"
+    And I wait "3" seconds
+    Then I should see current url contains "/dataset/100039/token/"
+    And I should see "Genomic data of the Puerto Rican Parrot (Amazona vittata) from a locally funded project."
```

Comment:
**question:** Is this test still `work in progress`? Because it has tag `@test`. And is this test necessary as supposedly the following scenario should be implemented:
```
Given I am on the dataset Admin page of any dataset with a status of anything EXCEPT "Published"
When I look at the bottom of the page
Then I should see a create/reset private URL button
And when I click that button it does create or reset the private URL for that dataset.
```
But in fact, the current existing scenario `Links to create mockup or to open mockup are always present for a non-published dataset` has covered it already.

Reply:
The tag will be removed, just used locally. 
the problem was with the submitted status (there was an error)
The published status is already tested. I just swapped the tests

Reply:
> the problem was with the submitted status (there was an error)

If that is the case, may be the test can be:
```
Scenario: Cannot change status to Submitted from non DataAvailableForReview
 Given I am on "/adminDataset/update/id/668"
  And should see "Private"
  And I should see "Create/Reset Private URL"
  And I select `"Submitted"` from the field "Dataset_upload_status"
  When I press the button "Save"
  Then I am on "/adminDataset/update/id/668"
  And I should see "Failed to update status!"

Scenario: Links to create mockup is present for a submitted dataset
   Given I am on "/adminDataset/update/id/5"
    And I should see "ImportFromEM"
    And I select "DataAvailableForReview" from the field "Dataset_upload_status"
    And I press the button "Save"
    And I should see "Updated successfully! "
    And I should see "Create/Reset Private URL"
    When I select "Submitted" from the field "Dataset_upload_status"
    And I press the button "Save"
    And  I am on "/adminDataset/update/id/5"
    And I should see "Create/Reset Private URL"
    And I press the button "Create/Reset Private URL"
    And I wait "3" seconds
    And I should see current url contains "/dataset/100039/token/"
    And I should see "Genomic data of the Puerto Rican Parrot (Amazona vittata) from a locally funded project."
```

Reply:
why do you want to add "Cannot change status to Submitted from non DataAvailableForReview". it's another PR

Reply:
Can you remind me which PR? If there is already a PR for it, you can ignore `Cannot change status to Submitted from non DataAvailableForReview"`

Reply:
not opened yet